www/Idfa.js

```diff
@@ -2,9 +2,9 @@ var exec = require("cordova/exec");
 var PLUGIN_NAME = "Idfa";
 
 module.exports = {
-    getInfo: function() {
-        return new Promise(function(resolve, reject) {
+    getInfo: function () {
```

Comment:
Please fix white spaces here and at line 6

Reply:
done